### DISCUSSION TITLE
How to update Lsp servers?

### DISCUSSION THREAD

**Discussion by arionrefat** [mason-org/mason.nvim#284] [Q&A] [answered]
before in `LspInstaller` i just had to press `U` in `LspinstallInfo` window?
how do I do that in Mason?

**Comment by williamboman** [comment, marked as answer]
Hello! In nvim-lsp-installer the `U` keybind would either simply reinstall all packages, or update outdated packages (assuming there are at least one package that has been flagged as outdated). The intended use case is the latter. In Mason it will only update packages that are flagged as outdated, so you will need to check for outdated packages first (press `C`).

As a side note: I've opened #285 which will make it so that outdated packages are automatically checked when opening the Mason window.

**Reply by arionrefat** [reply]
Thank you, appreciate the work ♥️
